cmd/restic/cmd_repair_snapshots.go

```diff
@@ -122,19 +122,19 @@ func runRepairSnapshots(ctx context.Context, gopts GlobalOptions, opts RepairOpt
 			node.Size = newSize
 			return node
 		},
-		RewriteFailedTree: func(_ restic.ID, path string, _ error) (restic.ID, error) {
+		RewriteFailedTree: func(_ restic.ID, path string, _ error) (restic.ID, *restic.SnapshotSummary, error) {
```

Comment:
How did that change from https://github.com/restic/restic/pull/5185 end up here?